integration_tests/pages/additionalConditionInput.ts

```diff
@@ -26,6 +27,33 @@ export default class AdditionalConditionsInputPage extends Page {
     return this
   }
 
+  enterTime = (): AdditionalConditionsInputPage => {
+    cy.get("input[name*='hour']").type('11')
+    cy.get("input[name*='minute']").type('30')
```

Comment:
What does the star do on name* ?

Reply:
Just a wildcard match on name as it appears?

Reply:
This jQuery selector matches any input where the name attribute contains some text. For example we have inputs called `appointmentTime[hour]`  `inductionDate[hour]` `appointmentTime[minute]`  `inductionDate[minute]` etc. This selector matches any input with hour/minute as a field

Reply:
https://api.jquery.com/attribute-contains-selector/